api/rate.go

```diff
@@ -26,12 +26,21 @@ func (a API) Rate(rate *models.Rate) (*models.RateReply, error) {
 }
 
 func (a API) rateRequest(rate *models.Rate) *models.Envelope {
-	rateRequestTypes := "LIST"
+	rateRequestTypes := "PREFERRED"
 	packageCount := 1
-	serviceType := "FEDEX_GROUND"
-	if rate.FromAddress.ShipsOutWithInternationalEconomy() {
-		serviceType = "INTERNATIONAL_ECONOMY"
+
+	// When the service type is smartpost, getting rates from FedEx API doesn't
+	// work
+	serviceType := rate.ServiceType()
+	serviceTypeInRequest := serviceType
+	if serviceType == "SMART_POST" {
+		// This is necessary. We can't get back smartpost rates. So using ground
+		// instead here.
+		// Per Page 239 of the Dev Guide: "Estimated shipping rates are not
+		// available for SmartPost Returns"
+		serviceTypeInRequest = "FEDEX_GROUND"
```

Comment:
If we don't need to include the smartpost details in our ground request we can just change the value of `serviceType` instead of creating a second variable.